src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertransactionsapi/integration/linkexpiry/MagicLinkExpiryIntegrationTest.kt

```diff
@@ -1,30 +1,24 @@
-package uk.gov.justice.digital.hmpps.prisonertransactionsapi.integration
+package uk.gov.justice.digital.hmpps.prisonertransactionsapi.integration.linkexpiry
 
 import com.nhaarman.mockitokotlin2.argumentCaptor
 import com.nhaarman.mockitokotlin2.doNothing
 import com.nhaarman.mockitokotlin2.eq
 import com.nhaarman.mockitokotlin2.verify
 import com.nhaarman.mockitokotlin2.whenever
 import org.junit.jupiter.api.Test
-import org.junit.jupiter.api.extension.ExtendWith
 import org.mockito.ArgumentMatchers.anyString
-import org.springframework.boot.test.mock.mockito.SpyBean
 import org.springframework.http.MediaType
 import org.springframework.test.context.TestPropertySource
 import org.springframework.web.reactive.function.BodyInserters
-import uk.gov.justice.digital.hmpps.prisonertransactionsapi.email.EmailSender
+import uk.gov.justice.digital.hmpps.prisonertransactionsapi.integration.IntegrationTestBase
 
-@ExtendWith(RedisExtension::class)
 @TestPropertySource(
   properties = [
     "secret.expiry=1s"
   ]
 )
 class MagicLinkExpiryIntegrationTest : IntegrationTestBase() {
```

Comment:
I've had to move this into its own package to make sure the tests run in the right order - for some as yet unknown reason the in memory Redis cache plays up when the tests run in the opposite order. Not at all impressed but don't want to get bogged down with this issue during an alpha phase. Hence sweeping it under the carpet - for now.